mongoCollections.ts

```diff
@@ -0,0 +1,18 @@
+import { dbConnection } from "./mongoConnection";
+
+// This will allow you to have one reference to each collection per app
+function getCollectionFn(collection: any) {
+	let _col: any = undefined;
```

Comment:
If we have multiple of these getCollectionFn calls, would _col stay defined from the first call when the second call is run? Did you test with multiple collections?

Reply:
I didn't test this time around because I've done this before, but it does work! See the formal explanation here if you're curious: https://docs.mongodb.com/manual/core/transactions/